### ISSUE TITLE
Unexpected `WriteReportingJavaFileObject` with annotation processor

### ISSUE THREAD

**Issue by regadas** [scalacenter/bloop#1114]
I see a couple of issues #1024 #956 #962 regarding this matter and now I'm, seeing it with https://github.com/google/auto.

@jvican Is there any workaround that i missed?

**Error** that I'm seeing in `metals.log`

```
@AutoValue processor threw an exception: com.sun.tools.javac.util.ClientCodeException: java.lang.IllegalArgumentException: Not supported: sbt.internal.inc.javac.WriteReportingJavaFileObject
```

**Comment by jvican** [comment]
Hi @regadas, thanks for reporting! What bloop version are you using? This is fixed in the latest versions.

**Comment by regadas** [comment]
Hi @jvican sorry for the delay ... so I was using the metals 0.76 bundled version which i believe it's `1.3.2`

**Comment by jvican** [comment]
Right, this is fixed in either 1.3.4 or 1.3.5. Please use latest Metals snapshot.

**Comment by regadas** [comment]
Cool! Thx @jvican !
